### PR TITLE
Update docs for contributing

### PR DESCRIPTION
**Pull request by ota-meshi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -331,9 +331,38 @@ Please use GitHub's Issues/PRs.
...
+- If you want to apply the same configuration to all the fixtures in that directory, add `_config.json`.
+
+To verify the output of invalid test cases requires `*-errors.json`, and `*-output.svelte` (for auto-fix). However, you don't have to add them yourself. If they do not exist, they will be automatically generated when you run the test. In other words, delete them manually when you want to recreate them.
+
+**Tips**:
+
+If you want to test only one rule, run the following command (for `indent` rule):
+
+```sh
+yarn mocha "tests/src/**/indent.ts" --reporter dot --timeout 60000
```
Comment (JounQin) [inline README.md:360]: Should `-g, --grep <pattern>            only run tests matching <pattern>` be preferred?

https://stackoverflow.com/questions/10832031/how-to-run-a-single-test-with-mocha

Comment (JounQin) [reply README.md:360]: The StackOverflow url could be added as reference.

Comment (ota-meshi) [reply README.md:360]: I find it difficult to use `-g` because it's hard for the ESLint rule tester to find out what to register in `it()`.

Comment (JounQin) [reply README.md:360]: 😅Fair enough.

**Review by JounQin — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ota-meshi) [reply README.md:360]: I misunderstood what you were saying.
It means to narrow down the test by the rule name. For example, `-g indent`.
I think it's easy to use, so I will change it to use `-g`.

**Comment by ota-meshi** [comment]
I changed the command. Could you see it fits your idea?

File: README.md [hunk 2]
```diff
@@ -331,9 +331,38 @@ Please use GitHub's Issues/PRs.
...
+To verify the output of invalid test cases requires `*-errors.json`, and `*-output.svelte` (for auto-fix). However, you don't have to add them yourself. If they do not exist, they will be automatically generated when you run the test. In other words, delete them manually when you want to recreate them.
+
+**Tips**:
+
+If you want to test only one rule, run the following command (for `indent` rule):
+
+```sh
+yarn test -g indent
+```
+
```
Comment (JounQin) [inline README.md:362]:
```suggestion

Take https://stackoverflow.com/questions/10832031/how-to-run-a-single-test-with-mocha as reference for details.

```

**Review by JounQin — CHANGES_REQUESTED** [review]

**Review by JounQin — APPROVED** [review]

**Pull request merged**